operations/configuration.md

```diff
@@ -102,6 +104,8 @@ Returns configuration of the enterprise and the client.
 | `PostalCode` | string | optional | Postal code. |
 | `CountryCode` | string | optional | ISO 3166-1 code of the [Country](configuration.md#country). |
 | `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude (output only, can't be set from API call) |
+| `Longitude` | number | optional | The longtitude (output only, can't be set from API call) |
```

Comment:
If this DTO is used both, for return values and also as parameters for e.g. update, the split it to two DTOs - AddressParameters and Address so that you can avoid this "output only" comment.

Also check the JSON example above, it's missing CountrySubdivisionCode and maybe I'd sort it there the same way as it is sorted in this table. This issue might be in other places of this documentation, so please check all address occurrences.

Reply:
Will fix this and open consolidation issue. These cases (different DTOs for input and output) should be handled consistently in whole documentation